en/includes/guides/account-configurations/login-security/sift-fraud-detection.md

```diff
@@ -25,64 +25,15 @@ To enable Sift fraud detection:
 
 1. On the Asgardeo Console, go to **Applciations**.
 2. Go to the **Login Flow** tab of the application and enable **Conditional Authentication**.
-3. Add a conditional authentication script. The following example conditional authentication script is for a scenario where:
-    - Authentication fails if the risk score exceeds 0.7.
-    - A second authentication step is triggered if the risk score falls between 0.5 and 0.7.
+3. Add a conditional authentication script.
```

Comment:
IMO, it'd be better to link to `Sample Conditional Authentication Scripts` section from here.

Reply:
Linked